library/cables/XLPE_500mm_132kV.yaml

```diff
@@ -1,7 +1,7 @@
 ac_resistance: 0.06
 capacitance: 200
 conductor_size: 500
-cost_per_km: 200000
+cost_per_km: 500000
```

Comment:
This should be added to a changelog. At this point it'd be good to have a section for this release for default assumption updates.